tiles/src/main/java/com/protomaps/basemap/layers/Boundaries.java

```diff
@@ -30,6 +30,7 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
       List<OsmReader.RelationMember<AdminRecord>> recs = sf.relationInfo(AdminRecord.class);
       if (recs.size() > 0) {
         OptionalInt minAdminLevel = recs.stream().mapToInt(r -> r.relation().adminLevel).min();
```

Comment:
NOTE: This looks like it prevents the City and County of San Francisco from appearing as a locality boundary line, instead it only appears as a county boundary line. An odd case, but this is true for several other "global cities" in the USA and elsewhere, too.